tests/conftest.py

```
@@ -10327,3 +10327,122 @@ def distribute_storage_classes_to_all_consumers():
+    ocs_storage_obj = OCP(
+        kind="storagecluster",
+        namespace=config.ENV_DATA["cluster_namespace"],
+        resource_name=constants.DEFAULT_STORAGE_CLUSTER,
+    )
+
+    initial_db_backup_info = ocs_storage_obj.get("ocs-storagecluster")["spec"][
+        "multiCloudGateway"
+    ]["dbBackup"]
+    log.info(f"Initial dbBackup value: {initial_db_backup_info}")
+
+    def factory(schedule_cron_interval, num_backups, snapshot_class):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `factory`

Add return type annotation: `None`

(ANN202)